terraform/application-plane/templates/hybrid-template.tf.template

```diff
@@ -1,8 +1,6 @@
-locals {}
-
 # It will deploy only the infraestructure of the siloed services
 module "hybrid_tenant___TENANT_ID__" {
   source = "git::__MODULE_SOURCE__//terraform//modules//tenant-apps?ref=v0.0.1"
   bucket_name = "__TENANT_ID__"
```

Comment:
Instead of Bucket name, how about creating a variable for tenant_id and use it internally in the module as bucket name, so the module is more generic because some apps might not have s3